data/governments.txt

```diff
@@ -516,8 +522,9 @@ government "Hai (Friendly Unfettered)"
 		"Pirate (Devil-Run Gang)" -.01
 		"Korath" -.01
 		"Hai" .01
+		"Hai (Unfettered)" .01
```

Comment:
not sure about this one

Reply:
I figured setting it same as their reaction to "Hai" for starters was best, but it would also make sense for it to be higher.

Reply:
no, I was thinking maybe it should not be there at all, meaning they wouldn't react when you attack the Unfettered (I don't think they should)

Reply:
Ah! I included it because it seems weird to me they would react to attacks against Hai in general but not Unfettered on at least the same level when they are themselves Unfettered. I could see someone attacking -enough- normal Unfettered placing these low enough to still be hostile even after Unfettered reputation resets, but wouldn't the best fix for that be to also include this in those resets?

If that does make sense, I could add that here alongside these changes as well. If not, I'll drop this back out then, yea, but I'd prefer to wait and give a chance for others to weigh in before going forward with either option, to avoid back and forth :)

Reply:
the Hai part is just there because they're fighting together (although that's wormhole access lol)

Reply:
```suggestion
		"Hai (Unfettered)" 1
```

Reply:
So to my understanding the "Friendly Unfettered" are just an Unfettered government type used when you need to use UHai as escorts without necessarily having the faction be peaceful with the player.
They're not narratively a different government, they're a hack for facilitating narrative.
That would mean that logically, anything you do which affects them should directly translate to affecting the regular UHai and vice versa.

This is my logic for the suggestion.
If they're being utilised differently to that though then I may be off-base and you can go whichever other way is appropriate.

Reply:
No, we definitely cant do that, or attacking the Unfettered in general makes you at war with both their civilian food managers and their HR appareances

Reply:
When such governments are used for narrative purposes, there's usually a reputation reset involved prior to the point that they're used.

If you're abandoning the plot in such a moment to go shoot at UHai, then you deserve to have the narrative get broken on you.

I'm not sure how these civilian food managers are handled, but if they're some kind of continuously present entity then they shouldn't be the government that's used for narrative purposes, they should be a separate one.

Reply:
Yeah they should be
I dont think there is a reputation reset, if we do this we need to make sure of it
Arent you're the one that made them the same government, iirc, in the hai shipyard PR? The aphid fleets?

Reply:
I thought that was a Unfettered Civilian gov, not a Friendly Unfettered gov?

Reply:
Do we need to better split it up? And add rep reset for the one in the HR mission line?

Reply:
There is no civ gov, yeah we shoud do that

Reply:
Okay, I have some other stuff lined up after my short shift tonight, but I'll try to get to this before bed today.

If either of you beats me to it with even a partial PR to the Evan h, I won't be upset though ;)

Reply:
done, and merged, I implemented this change there too, so you can resolve this